src/GraphEditor.jsx

```diff
@@ -12,6 +12,14 @@ import JXGBoard from './JXGBoard';
 import {displayGraphType, handleFormUpdate, getCohortId} from './utils';
 
 export default class GraphEditor extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            gLinearDemandSupplyAUCArea: []
```

Comment:
I would change this to:
```
this.state = {
    areaA: null,
    areaB: null,
    areaC: null
};
```

The `g` prefix is only used for values that are saved to the back-end, sorry that's not really mentioned anywhere, just kind of a standard I'm using for this project.

And also, I just find it easier to deal with state values directly - these don't need to be in an array.

Reply:
Ok, got it regarding `g`.